src/constraint_functions.jl

```diff
@@ -59,22 +66,31 @@ If you create new capacity variables, it is beneficial to include as well a meth
 function and the corresponding node types.
 """
 function constraints_capacity_installed(m, n::Node, 𝒯::TimeStructure, modeltype::EnergyModel)
-
-    cap = capacity(n)
-    @constraint(m, [t ∈ 𝒯],
-        m[:cap_inst][n, t] == cap[t]
-    )
+    # Fix the installed capacity to the upper bound
+   for t ∈ 𝒯
+        fix(m[:cap_inst][n, t], capacity(n, t); force=true)
+   end
 end
 function constraints_capacity_installed(m, n::Storage, 𝒯::TimeStructure, modeltype::EnergyModel)
+    # Extract the required fields from the composite type
+    par_charge = charge(n)
```

Comment:
what is the mnemonic for the `par_xxx` variables, parameter?

Reply:
The initial aim was to reduce the number of function calls. However, thinking a bit more about it, it really does not make too much of a difference, Hence, I removed it.